<launch>
  <node name="decision_maker" pkg="crowdbot_active_slam"
        type="decision_maker" output="screen">
    <param name="primitive_filename"
           value="$(find crowdbot_active_slam)/matlab/mprim/pioneer.mprim"/>
   <!-- Choose exploration type:
        shortest_frontier, utility_standard or utility_normalized -->
    <param name="exploration_type" value="utility_standard"/>
  </node>
</launch>
